packages/shopping-cart/src/types.ts

```diff
@@ -957,6 +957,7 @@ export interface RequestCartProductExtra extends ResponseCartProductExtra {
 	headstart_theme?: string;
 	feature_slug?: string;
 	is_hundred_year_domain?: boolean;
+	flow_name?: string;
```

Comment:
I guess this differs from `signup_flow` because it might not be for signup. Still, it's easy to conflate these.

Could you add a comment to this property to explain its purpose?